vuepress/docs/next/docs/getting-started/development-concepts.md

```diff
@@ -0,0 +1,59 @@
+
+Entando provides a composable environment where these building blocks can be built independently, enabling parallel teams to accelerate development and innovation on a cloud and Kubernetes-native stack.
+
+![Entando Platform Development Process](./img/concepts.png)
+
+## Create Components 
+The first step in building a modern application on Entando is to design and create each independant component or functional block for your application. Developers can use the Entando Component Generator or Blueprint to automate the building and packaging of project files, with the flexibility to customize. In this way, new business capabilities can be integrated into your application without rewriting existing systems. 
+
+| Documentation | Tutorials        |
+| :------------ | :--------- |
+| [Entando Component Generator](../create/component-gen-overview.md) | [Micro Frontends](../../tutorials/create/mfe/) |
+| [Entando Blueprint Feature](../create/blueprint-features.md) | [Microservices](../../tutorials/create/ms/generate-microservices-and-micro-frontends.md) |
+| [Component Generation Technologies](../create/component-gen-tech.md) | [Build and Publish Project Bundles](../../tutorials/create/pb/publish-project-bundle.md) |
+
+
+	
+## Curate and Share Components 
+On the Entando Platform, Curate and Share means developer teams can share components within a central catalog made available to multiple teams or organizations. Entando's distributed and composable development process allows you to quickly edit, test, analyze, and publish a singular component or packaged business capabilities (PBC) repeatedly, without touching other parts of your application. This includes the ability to publish content and pages along with functional components such as micro frontends and microservices. 
+
+| Documentation | Tutorials        |
+| :------------ | :-------- |
+| [Bundle and Component Descriptors](../curate/ecr-bundle-details.md) | [Install Bundles from a Private Git Repository](../../tutorials/curate/ecr-private-git-repo.md) |
+| [Filtering Bundles](../curate/ecr-bundle-filters.md) | [Install Bundle Plugins from a Private Image Repo](../../tutorials/curate/ecr-private-images.md) |
+| [Customize Bundle Info in App Builder](../curate/ecr-bundle-presentation-config.md) |   |
+| [Bundle Version and Updates FAQ](../curate/ecr-bundle-versions-faq.md) |  |
+| [Bundle Uninstall](../curate/ecr-uninstall-flow.md) |  |
+
+
+## Compose an Application 
+Business IT groups and developers Compose an Application with the Entando App Builder, a low-code composition UI. Assemble or update an application from the components already built in step one, adding templates, widgets, or full PBCs for functionality. The Entando Component Manager and Repository are integrated into the App Builder, making the composable process flexible and agile by managing all your components there. 
```

Comment:
agree with leading with target user, as is done in revised Curate and Share description

would change Business IT groups to Business units; BU is a common acronym used in this context

should use full name of Entando Component Repository for consistent naming convention and to clarify that the Manager and Repository are two separate entities; also add links

sample rephrasing: "Business units and developers Compose an application with the Entando App Builder, a low-code composition UI. Assemble or update an application from pre-built components, including templates, widgets and PBCs to add functionality. The Entando Component Manager and Entando Component Repository are integrated into the App Builder for flexible and agile component management."